semcore/time-picker/src/PickerInput.jsx

```diff
@@ -231,15 +219,6 @@ class Minutes extends ItemPicker {
       }
     }
     if (event.key === 'Enter') this.submitChanges(event);
-    if (event.key === 'ArrowRight') {
```

Comment:
Why did you remove right arrow handling but preserved left arrow handling?

Reply:
It didn't work. This code is for "minutes" part and should have only left handle (to the hours) as in hours there is only right handle (to the minutes)